client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/useInteractionsTab.ts

```diff
@@ -108,12 +108,50 @@ const useInteractionsTab = (props: useInteractionsTabInput) :  useInteractionsTa
         loadInteractionById(addedInteraction.id as number);
     }
 
+    const handleDeleteContactEvent = (contactEventId: number) => {
+        Swal.fire({
+            icon: 'warning',
+            title: 'האם אתה בטוח שתרצה למחוק את האירוע?',
+            text: 'שים לב, בעת מחיקת האירוע ימחקו כל המגעים שנכחו בו',
+            showCancelButton: true,
+            cancelButtonText: 'בטל',
+            cancelButtonColor: theme.palette.error.main,
+            confirmButtonColor: theme.palette.primary.main,
+            confirmButtonText: 'כן, המשך',
+            customClass: {
+                title: classes.swalTitle,
+                content: classes.swalText
+            }
+        }).then((result) => {
+            if (result.value) {
+                axios.delete('/intersections/deleteContactEvent', {
+                    params: {contactEventId}
```

Comment:
https://stackoverflow.com/questions/51069552/axios-delete-request-with-body-and-headers

אולי אפשר ככה להעביר את זה בBODY

Reply:
data הייתה האופציה הראשונה שמצאתי, ולא מצאתי איך לשלוף בלקוח
מה הבעיה עם params?

Reply:
שיניתי שזה ישלח data במקום Params אבל זה עדיין תופס עם query

Reply:
כי זה טיפה יותר מאובטח שזה בbody ולא בURL